packages/ods/src/components/datepicker/tests/rendering/ods-datepicker.e2e.ts

```diff
@@ -139,4 +141,55 @@ describe('ods-datepicker rendering', () => {
       expect(await isInErrorState()).toBe(true);
     });
   });
+
+  describe('isClearable', () => {
+    it('should render a clearable button', async() => {
+      await setup('<ods-datepicker is-clearable></ods-datepicker>');
+      await page.evaluate(() => {
+        document.querySelector<OdsDatepicker & HTMLElement>('ods-datepicker')!.value = new Date('10 May 2024');
+      });
+      await page.waitForChanges();
+
+      buttonClearable = await page.find('ods-datepicker >>> ods-button[icon="xmark"]');
+      expect(buttonClearable).not.toBeNull();
+    });
+
+    it('should render a disabled clearable button when datepicker is disabled', async() => {
+      await setup('<ods-datepicker is-disabled is-clearable></ods-datepicker>');
+      await page.evaluate(() => {
+        document.querySelector<OdsDatepicker & HTMLElement>('ods-datepicker')!.value = new Date('10 May 2024');
+      });
+      await page.waitForChanges();
+
+      buttonClearable = await page.find('ods-datepicker >>> ods-button[icon="xmark"]');
+      expect(buttonClearable.getAttribute('is-disabled')).toBe('');
+    });
+
+    it('should render a disabled clearable button when datepicker is readonly', async() => {
+      await setup('<ods-datepicker is-readonly is-clearable></ods-datepicker>');
+      await page.evaluate(() => {
+        document.querySelector<OdsDatepicker & HTMLElement>('ods-datepicker')!.value = new Date('10 May 2024');
+      });
+      await page.waitForChanges();
+
+      buttonClearable = await page.find('ods-datepicker >>> ods-button[icon="xmark"]');
+      expect(buttonClearable.getAttribute('is-disabled')).toBe('');
+    });
+
+    it('should render clearable button when type wrong format', async() => {
+      await setup('<ods-datepicker is-clearable ></ods-datepicker>');
+
+      await el.type('clearable');
+      await page.waitForChanges();
+
+      buttonClearable = await page.find('ods-datepicker >>> ods-button[icon="xmark"]');
+      expect(buttonClearable).not.toBeNull();
+    });
+
+    it('should not render clearable button when value wrong format', async() => {
```

Comment:
actually it should, that's the whole point of the change, that the clearable button appear as soon as we type at least one character in the field, regarding its validity